templates/circom/circuit.circom

```diff
@@ -0,0 +1,23 @@
+pragma circom 2.0.8;
+
+// You can include files from `circomlib` like this.
+include "node_modules/circomlib/circuits/comparators.circom";
+
+template templateTemplateName() {
```

Comment:
I tried the `init` command before reading any of the code, and I found the "Circom template name" question opaque.  Circom code can have many template definitions linking together in arbitrary ways.  So when I answered that question I had a feeling I was signing up for something more than I understood.  My suggestion is to just eliminate that question from the I/O and change this from a templated variable to: 
```suggestion
template Main() {
```

Reply:
Sounds good. I made all of the related changes in [65d5995](https://github.com/Sindri-Labs/sindri-js/pull/24/commits/65d59952ea5b3e22ca443c029d0b686a8230cb45).